keyloader/keyloader.go

```diff
@@ -333,6 +350,35 @@ func singleKeyIdentifier(store *configstore.Store) (string, error) {
 	return "", errors.New("ambiguous config: several encryption keys found and no identifier supplied")
 }
 
+func newKey(cfg *KeyConfig) (symmecrypt.Key, error) {
+	factory, err := symmecrypt.GetKeyFactory(cfg.Cipher)
+	if err != nil {
+		return nil, err
+	}
+
+	if cfg.Sealed {
+		return newSealedKey(cfg, factory), nil
```

Comment:
newSealedKey should also be patched to support sequential keys ?

Reply:
also, the KeyConfig fields are all used in the sealed keys extra data (AEAD) to make sure they're not altered (to avoid altering the behavior). This should be taken into consideration if we are to add new fields.